// Copyright 2011 Yunrang Inc. All Rights Reserved.
// Author: jianxu@yunrang.com (Jian Xu)

namespace cpp util_mysql

struct TestStruct {
  1: optional list<i32> data_int32,
  2: bool data_bool,
  3: double data_double,
  4: string data_str,
  5: i64 data_int64,
}
